spec/importer_spec.rb

```diff
@@ -0,0 +1,178 @@
+require 'spec_helper'
+require 'ronin/masscan/importer'
+require 'masscan/output_file'
+require 'ronin/db'
+
+RSpec.describe Ronin::Masscan::Importer do
+  let(:output_file) { instance_double('Masscan::OutputFile') }
+
+  describe '.import_file' do
+    let(:path) { '/path/to/file' }
+
+    before do
+      allow(Masscan::OutputFile).to receive(:new).with(path).and_return(output_file)
+    end
+
+    it 'must initialize Masscan::OutputFile and import from it' do
+      expect(subject).to receive(:import).with(output_file)
+
+      subject.import_file(path)
```

Comment:
We should probably test `.import` and `.import_file` end-to-end. You can initialize a test database using `Ronin::DB.connect('sqlite3::memory')` in a `before(:all) { ... }` block. Then add `Ronin::DB::OpenPort.destroy_all`, etc, to a `after do ... end` block to delete any imported records.